frontend/src/components/common/DataTableHeader.vue

```diff
@@ -0,0 +1,149 @@
+<script setup lang="ts">
```

Comment:
Hi Jota, this file is duplicated, you have one here, another one under managePermissions folder. I think this DataTableHeader component and the ConfirmDialogText component can both placed under the managePermissions folder, cause they are only used for that page. 

And under managePermissions folder, we can create a new directory maybe called table, and move all the table related ones under it.

Reply:
that's a good idea! I'll remove the duplicate and make a folder for the table ralated